packages/amplify_authenticator/lib/src/mixins/authenticator_date_field.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:amplify_authenticator/src/theme/amplify_theme.dart';
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+mixin AuthenticatorDateField<FieldType,
+        T extends AuthenticatorFormField<FieldType, String, T>>
+    on AuthenticatorFormFieldState<FieldType, String, T> {
+  String _convertToDateString(DateTime dt) {
+    return "${dt.year.toString()}-${dt.month.toString().padLeft(2, '0')}-${dt.day.toString().padLeft(2, '0')}";
```

Comment:
The `intl` package has a date formatter. You should be able to do something like 
```dart
import 'package:intl/intl.dart';

final DateFormat formatter = DateFormat('yyyy-MM-dd');

mixin AuthenticatorDateField ... {
  String _convertToDateString(DateTime dt) {
    return formatter.format(dt);
  }
```
}